cif_pow.dic

```diff
@@ -1494,10 +1497,18 @@ save_
 save_pd_block.id
 
     _definition.id                '_pd_block.id'
+    loop_
+      _definition_replaced.id
+      _definition_replaced.by
+         1                        '_pd_phase.id'
+         2                        '_pd_diffractogram.id'
     _alias.definition_id          '_pd_block_id'
-    _definition.update            2023-01-06
+    _definition.update            2023-06-08
     _description.text
 ;
+    **DEPRECATED**
+    Use _pd_phase.id or _pd_diffractogram.id, as necessary.
+
```

Comment:
`_pd_block.id` is not really replaced by those other data names. `_pd_block.id` identifies a data block, they identify phases and diffractograms. If anything, it is replaced by `_audit.block_code`.

Reply:
Strictly, yes, but, the data items that were linked to _pd_block.id were only ever used to identify phases or diffractograms.

Reply:
OK, it might still confuse some people thinking that a `_pd_block.id` identifies a block, whereas the other `ids` identify something completely different. We might need to expand the explanation.